api/nakadi-event-bus-api.yaml

```diff
@@ -1567,13 +1651,14 @@ definitions:
           future there could be others.
       schema:
         type: string
-        $ref: '#/definitions/EventTypeSchema'
         description: |
           The schema as string in the syntax defined in the field type. Failure to respect the
           syntax will fail any operation on an EventType.
 
-          To have a generic, undefined schema it is possible to define the schema as `"schema":
-          "{\"additionalProperties\": true}"`.
+          When defining a schema, every property should be explicitly defined with its type. It means that,
+          additionalProperties true is not accepted.
```

Comment:
@conorclifford correct. I would be possible to use `additionalProperties` to define a map of some specific type, but once defined, it won't be possible to change it. So if additionalProperties is used within an object, this object is closed for schema changes, since additionalProperties means it's extensible.